basis/basis.py

```diff
@@ -0,0 +1,172 @@
+import abc
```

Comment:
This file should be in `smolyay/basis.py` so that it can be imported like:
```py
import smolyay.basis
```
You should `git mv` this file, then `git rm` the `basis/` directory and `__init__.py`.